package.json

```diff
@@ -721,7 +721,7 @@
                 "background": {
                     "activeOnStart": true,
                     "beginsPattern": "^.*(Job host stopped|signaling restart).*$",
-                    "endsPattern": "^.*Host lock lease acquired by instance ID.*$"
+                    "endsPattern": "^Application started.+$"
```

Comment:
@JustinGrote not sure if you remember, but we already discussed this [here](https://github.com/microsoft/vscode-azurefunctions/issues/1245#issuecomment-489176470).

This may work for some languages, but not all. Specifically Python would fail unless we waited for the lock lease message. See https://github.com/Microsoft/vscode-azurefunctions/issues/660 

I've asked the functions team to give us a debug-specific message that will work in all languages. See https://github.com/Azure/azure-functions-host/issues/4384

Reply:
The message changed so I thought this got resolved but apparently not. My bad :) guess this is stuck on that request.

Reply:
If you want to dig deeper and find out exactly when/why/how the message changed that would be great, but yeah at the moment I'm just waiting on that issue